### ISSUE TITLE
Pressing Play does nothing EGL2 closes when pressing verify

### ISSUE THREAD

**Issue by 1tuffNutzgFNBR** [WorkingRobot/EGL2#21]
Not sure what has happened the old version I had no problems with was able to use the program and now the updated version doesn't seem to work for me i have attached log file 
[2020-06-14_09-43-26.log](https://github.com/WorkingRobot/EGL2/files/4775688/2020-06-14_09-43-26.log)

**Comment by 1tuffNutzgFNBR** [comment]
This may of been my problem may have had an issue with the install location in the settings will keep you updated

**Comment by Bevan0** [comment]
Make sure your compression settings are the same ones you used on your previous install. If this doesn't change anything, make sure your game directory exists (you set this in Setup)

**Comment by 1tuffNutzgFNBR** [comment]
Yeah it was pointing to the wrong folder now it created a folder called workaround and installed in there, Quick question how do updates work now i use to be able to click update but now there is no button for it just verify which takes a little longer doing it that way, is it just auto-detected or something?
